src/core/org/apache/jmeter/resources/messages_fr.properties

```diff
@@ -853,7 +853,7 @@ reportgenerator_summary_statistics_kbytes=Ko/sec
 reportgenerator_summary_statistics_label=Libell\u00E9
 reportgenerator_summary_statistics_max=Max
 reportgenerator_summary_statistics_min=Min
-reportgenerator_summary_statistics_percentile_fmt=%d%% Ligne
+reportgenerator_summary_statistics_percentile_fmt=%de percentile
```

Comment:
In JMeter (aggregate graph, aggregate report), we use "centile", so it's will better to continue to use the word.